flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyMessage.java

```diff
@@ -539,4 +542,60 @@ static CloseRequest readFrom(@SuppressWarnings("unused") ByteBuf buffer) throws
 			return new CloseRequest();
 		}
 	}
+
+	static class AddCredit extends NettyMessage {
```

Comment:
Please add a comment ("incremental credit announcement from the client to the server"?).